Sources/BraintreeCore/BTConfiguration+Extensions.swift

```diff
@@ -37,13 +37,6 @@ import PassKit
         json?["payWithVenmo"]["environment"].asString()
     }
 
-    // MARK: - BTConfiguration+UnionPay
-
-    /// Indicates whether UnionPay is enabled for the merchant account.
-    var isUnionPayEnabled: Bool {
```

Comment:
Since the properties in this extension are public should this go in the CHANGELOG?

Reply:
Good question. I was thinking since because in V5, this lived in the UnionPay module ([here](https://github.com/braintree/braintree_ios/blob/fdb5561f47d912e6629dc0ec500d4155958f84b1/Sources/BraintreeUnionPay/Public/BraintreeUnionPay/BTConfiguration%2BUnionPay.h#L13-L16)), that the CHANGELOG entry for nuking the whole module would cover it.

But you're right that our CHANGELOG is keeping track across the betas, so to follow that pattern we should probably call it out. Will update!

Reply:
Updated